conf.py

```diff
@@ -40,7 +40,7 @@
 
 # General information about the project.
 project = u'Gisquick'
-copyright = u'2016-2019, Gisquick team'
+copyright = u'2016-2019, Gisquick team | Münzberger Josef, Usik Svetlana, Vaňková Zuzana'
```

Comment:
Do not modify footer. Add a new section to "Other", eg. "Authors (of this documentation)". Mention that major upgrade of this documentation (Gisquick 1 -> 2) has been done within FGIS project.

Reply:
Also update 2019 to 2020.